pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -28,7 +28,8 @@ func (client *GrafanaComClient) GetPlugin(pluginId, repoUrl string) (models.Plug
 	body, err := sendRequestGetBytes(HttpClient, repoUrl, "repo", pluginId)
 	if err != nil {
 		if errors.Is(err, ErrNotFoundError) {
-			return models.Plugin{}, errutil.Wrap("Failed to find requested plugin, check if the plugin_id is correct", err)
+			return models.Plugin{}, errutil.Wrap(
+				fmt.Sprintf("Failed to find requested plugin, check if the plugin_id: %s is correct", pluginId), err)
```

Comment:
The colon makes the sentence difficult to understand. I think the following makes for better language. WDYT?
```suggestion
				fmt.Sprintf("Failed to find requested plugin, check if the plugin_id (%s) is correct", pluginId), err)
```